app/components/pages/SitesDetailPage.js

```diff
@@ -37,6 +37,14 @@ class SitesPage extends React.Component {
                       </div>
                       <div className="stats">
                         <div className="list">
+                          <div className="item">
+                            <div className="label">
+                              Type
+                            </div>
+                            <div className="value">
+                              {this.props.type === 'iba' ? 'IBA' : 'Critical Site'}
```

Comment:
Could we use the I18n key, as it has translations for other languages.

Reply:
Ok. I see key `iba` and `csn`, should the type be `csn` or `Critical Site`?

Reply:
Critical Site.

Reply:
Done.